microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -20,44 +20,116 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+<!-- ------------------------------ -->
+#### Controlling whether typed characters are sent
+
+You can control whether typed characters are sent to your default search provider to provide search and site suggestions.  To enable or disable sending your typed characters in the Address bar to your default search provider:
+
+1. Select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**.<!--edge://settings/privacy-->
+
+1. In the **Services** section, select **Address bar and search** > **Search suggestions and filters**,<!--edge://settings/searchFilters--> and then turn on the **Show me search and site suggestions using my typed characters** toggle:
+
+   ![The toggle in Edge Settings for "Show me search and site suggestions using my typed characters"](./index-images/search-suggestions-typed-chars.png)
+
+   If you turn off that toggle, your typed characters are no longer sent to your default search provider, but your search queries are still sent to your default search provider, to provide search results.<!--per ui: "If you turn this off, you'll only see suggestions from your favorites and history. Typed characters won't be sent to the search engine used in the address bar." -->
+
+
+<!-- ------------------------------ -->
+#### Microsoft Search in Bing
+
+If you're signed into Microsoft Edge using your work or school account (Microsoft Entra ID account), your typed-in characters are sent to Microsoft Search in Bing, in addition to being sent to your default search engine.  As you type, Microsoft Search in Bing returns work- or school-related results and cross-device suggested queries to you.
+
+Your administrator can choose to turn on or off the work- or school-related results or query suggestions entirely for your organization, by using the `AddressBarMicrosoftSearchInBingProviderEnabled` Microsoft Search in Bing (MSB) policy.  If your administrator turns off the `AddressBarMicrosoftSearchInBingProviderEnabled` policy:
+* You don't see work- or school- related results from Microsoft Search in Bing.
+* You don't see query suggestions from other devices on which you installed Microsoft Edge.  (Regardless of the policy setting, you see query suggestions from your default search engine that you previously searched on the present device.)
+
+To further protect your privacy, you can keep the suggested-queries function, without sending information about the suggested queries to Microsoft.  This data is never associated with you or your device, even if you turn on this setting.  To prevent sending information about the suggested queries to Microsoft, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**, and then in the **Search and service improvement** section, turn off the toggle **Help improve Microsoft products by sending the results from searches on the web**.
```

Comment:
I don't think we need to first sentence about protecting your privacy, as this information is already very low privacy risk and isn't associated with you or your device. Keeping this sentence makes it seem like its cause for privacy concern.

Reordered and reworded for clarity. Changed never to not to reduce privacy risks. 

```suggestion
To prevent sending information about suggested queries to Microsoft, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**, and then in the **Search and service improvement** section, turn off the toggle **Help improve Microsoft products by sending the results from searches on the web**. Even if you turn on this setting, information about suggested queries is not associated with you or your device. 
```

Also, this section should be moved elsewhere, outside of the Microsoft Search in Bing section. This seems to apply to all cases where suggested queries are returned, not just to Microsoft Search in Bing for enterprises

Reply:
result; also added heading & reworded "even if":
```
<!-- ------------------------------ -->
#### Preventing sending information about suggested queries

To prevent sending information about suggested 
queries to Microsoft, select **Settings and more** 
(![Settings and more icon](./index-images/settings-
and-more-icon.png)) > **Settings** > **Privacy, 
search, and services**, and then in the **Search and 
service improvement** section, turn off the toggle **Help 
improve Microsoft products by sending the results from 
searches on the web**.  Even if this setting is on, information 
about suggested queries is not associated with you or your 
device.
```